discovery-package/src/main/resources/bin/start.sh

```diff
@@ -97,32 +93,30 @@ else
   nonStrictVerifySslCertificatesOfServices=true
 fi
 
-if [ "$(uname)" = "OS/390" ]
-then
-    QUICK_START=-Xquickstart
+if [ "$(uname)" = "OS/390" ]; then
+    QUICK_START="-Xquickstart"
 fi
 
 # setting the cookieName based on the instances
-
+ZWE_components_gateway_apiml_security_auth_uniqueCookie="${ZWE_components_gateway_apiml_security_auth_uniqueCookie:-false}"
 if [ "${ZWE_components_gateway_apiml_security_auth_uniqueCookie}" = "true"]; then
```

Comment:
This was showing in the logs an error because the resulting expression in z/OS shell was `if [ = true ]` which is incorrect and was showing an error in logs